packages/common/src/types.ts

```diff
@@ -414,8 +414,151 @@ export enum FileExtension {
+ */
+export const ItemTypes = [
+  "API Key",
+  "Layer Template",
+  "360 VR Experience",
+  "Apache Parquet",
+  "AppBuilder Widget Package",
+  "Desktop Add In",
+  "Explorer Add In",
+  "Explorer Map",
+  "Explorer Map",
+  "Explorer Layer",
+  "Windows Mobile Package",
+  "ArcGIS Pro Add In",
+  "ArcGIS Pro Configuration",
+  "Globe Document",
+  "Map Document",
+  "Map Document",
+  "ArcPad Package",
+  "Published Map",
+  "Scene Document",
+  "CityEngine Web Scene",
+  "Code Sample",
+  "CSV Collection",
+  "CSV",
+  "CAD Drawing",
+  "Deep Learning Package",
+  "Desktop Application",
+  "Desktop Application Template",
+  "Desktop Style",
+  "Earth Configuration",
+  "Esri Classifier Definition",
+  "Feature Collection",
+  "File Geodatabase",
+  "Form",
+  "GeoJson",
+  "Geoprocessing Package",
+  "GeoPackage",
+  "Geoprocessing Sample",
+  "GML",
+  "Image Collection",
+  "Image",
+  "iWork Keynote",
+  "iWork Numbers",
+  "iWork Pages",
+  "KML Collection",
+  "KML",
+  "Knowledge Graph",
+  "Layer",
+  "Layer Package",
+  "Layout",
+  "Locator Package",
+  "Map Package",
+  "Map Template",
+  "Microsoft Excel",
+  "Microsoft Powerpoint",
+  "Visio Document",
+  "Microsoft Word",
+  "Mobile Basemap Package",
+  "Mobile Map Package",
+  "Mobile Scene Package",
+  "Notebook",
+  "PDF",
+  "Pro Map",
+  "Pro Project",
+  "Pro Report",
+  "Project Package",
+  "Project Template",
+  "Raster function template",
+  "Rule Package",
+  "Scene Package",
+  "Service Definition",
+  "Shapefile",
+  "Survey123 Add In",
+  "Tile Package",
+  "Vector Tile Package",
+  "Tile Layer",
+  "Workflow Manager Package",
+  "Document Link",
+  "Feature Service",
+  "Geocoding Service",
+  "Geodata Service",
+  "Geometry Service",
+  "Geoprocessing Service",
+  "Geoenrichment Service",
+  "GeoRSS",
+  "Globe Service",
+  "Image Service",
+  "Map Service",
+  "Network Analysis Service",
+  "Image Service",
+  "Vector Tile Service",
+  "Video Service",
+  "WFS",
+  "WMS",
+  "WMTS",
+  "OGCFeatureServer",
+  "Scene Service",
+  "Stream Service",
+  "Workflow Manager Service",
+  "Image Service",
+  "Web Mapping Application",
+  "Web Map",
+  "Web Scene",
+  "Web Experience Template",
+  "Mobile Application",
+  "AppBuilder Extension",
+  "Google Drive",
+  "Dropbox",
+  "OneDrive",
+  "Map Service",
+  "StoryMap",
+  "Dashboard",
+  "Hub Initiative",
+  "Hub Site Application",
+  "Web Experience",
+  "Insights Workbook Package",
+  "Application",
+  "ArcGIS Explorer Application Configuration",
+  "ArcMap Document",
+  "Layer File",
+  "Solution",
+  "Spatial Database Engine",
+  "Compact Tile Package",
+  "netCDF",
+  "datastore catalog service",
+  "Pro Report Template",
+  "Real Time Analytic",
+  "Big Data Analytic",
+  "Data Store",
+  "Big Data File Share",
+  "Hub Page",
+  "Hub Project",
+  "Code Attachment",
+  "Solution",
+  "Hub Initiative Template",
+  "Quickcapture Project",
+]
+
```

Comment:
Once https://devtopia.esri.com/dc/hub-search/pull/159 merges, you can remove this from this PR and just leave the comment on the enum below.

Reply:
Removed the array and just left the comment on enum.